example-apps/dashnote-starter/README.md

```
@@ -0,0 +1,106 @@
+
+```sh
+npm install
+npm run dev
+```
+
+Open the printed URL, paste the mnemonic, and you should see your (empty) note
+list.
+
+## What's in here
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)